app/tests/forms/test_submission_integration.py

```diff
@@ -128,20 +133,50 @@ def test_cannot_create_event_form_evaluation_submission_if_not_attended(member_c
 
 
 def test_post_submission_returns_http_status_201_created(
-    member_client, form, submission, answer
+    member_client, event_form, submission, answer
+):
+    url = _get_submission_url(answer.submission.form)
+
+    response = member_client.post(
+        url, data=_create_submission_data(event_form.fields.first())
+    )
+
+    assert response.status_code == status.HTTP_201_CREATED
+
+
+def test_post_submission_to_event_form_can_overwrite_submission_if_unregistered(
+    member_client, event_form, submission, answer
 ):
     url = _get_submission_url(answer.submission.form)
 
+    member_client.post(url, data=_create_submission_data(event_form.fields.first()))
     response = member_client.post(
-        url, data=_create_submission_data(form.fields.first())
+        url, data=_create_submission_data(event_form.fields.first())
     )
 
     assert response.status_code == status.HTTP_201_CREATED
 
 
+def test_post_submission_to_event_form_cant_overwrite_submission_if_registered(
+    member, member_client, event_form
+):
+    # TODO: fix these factories (the users must be the same)
+    submission = SubmissionFactory(user=member, form=event_form)
+    answer = AnswerFactory(submission=submission, field=event_form.fields.first())
+    RegistrationFactory(event=event_form.event, user=member)
+    url = _get_submission_url(answer.submission.form)
+
+    member_client.post(url, data=_create_submission_data(event_form.fields.first()))
+    response = member_client.post(
+        url, data=_create_submission_data(event_form.fields.first())
+    )
+
+    assert response.status_code == status.HTTP_409_CONFLICT
```

Comment:
Hvor kommer 409 fra?

Reply:
409 kommer fra at en submission for eventet finnes allerede. (unique_together på form og user)